hack/test-net-e2e.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
This should be an extended test, in test/extended/network.sh and follow the patterns there.